client/landing/stepper/declarative-flow/hundred-year-plan.ts

```diff
@@ -68,7 +104,44 @@ const HundredYearPlanFlow: Flow = {
 		function submit( providedDependencies: ProvidedDependencies = {} ) {
 			recordSubmitStep( providedDependencies, '', flowName, _currentStep );
 
+			const updateCartForExistingSite = async () => {
+				if ( ! providedDependencies?.selectedSite ) {
+					return;
+				}
+				const site: SiteDetails = providedDependencies.selectedSite as SiteDetails;
+				const siteSlug = new URL( site.URL ).host;
+				if ( ! siteSlug ) {
+					return;
+				}
+				const productsToAdd = [
+					{
+						product_slug: PLAN_100_YEARS,
+					},
+				];
+				if ( providedDependencies?.selectedDomain ) {
```

Comment:
@aneeshd16 I'm not sure how I can run through a flow that covers this part of the code since there is no domain step in the existing site flow. Could you help me clarify?

Reply:
You can test this by choosing an existing site with a custom bundled domain. Here's an example:

https://github.com/Automattic/wp-calypso/assets/5436027/c2848b67-958f-4612-b950-a91041379e77